src/components/engagement/engagement.service.ts

```diff
@@ -777,7 +777,7 @@ export class EngagementService {
   async readOne(
     id: string,
     session: ISession
-  ): Promise<LanguageEngagement | InternshipEngagement> {
+  ): Promise<LanguageEngagement | InternshipEngagement | any> {
```

Comment:
```suggestion
  ): Promise<LanguageEngagement | InternshipEngagement> {
```

Reply:
yep I missed that on the first commit, fixed